docs/dictionary.rst

```diff
@@ -0,0 +1,1101 @@
+.. This file was created using GreatGloss. It is highly recommended to update the
```

Comment:
It shouldn't be `highly recommended`, it should be marked as a generated file with "DO NOT UPDATE" as a comment, or it should always be updated manually.